tests/test_validators.py

```diff
@@ -19,14 +20,23 @@ def validate(self, value):
             max_length=self.max_length,
         )
 
+    def validate_subtask_price_value(self, value):
+        return validators.validate_integer(
+            field_name=self.field_name,
+            value=value,
+        )
+
     def test_value_too_long(self):
         with self.assertRaises(exceptions.FieldError):
             self.validate('*'*(self.max_length+1))
 
     def test_value_valid(self):
         for x in range(self.max_length+1):
             self.validate('*'*x)
+        self.validate_subtask_price_value(fake.pyint())
```

Comment:
this should be a separate test method ... though, generally, as mentioned above, it should be a separate test case anyway...